app/models/draft_content_item.rb

```diff
@@ -35,6 +35,29 @@ class DraftContentItem < ActiveRecord::Base
   }
   validates_with RoutesAndRedirectsValidator
 
+  validates :description, well_formed_content_types: { must_include: "text/html" }
+  validates :details, well_formed_content_types: { must_include: "text/html" }
+
+  # Postgres's JSON columns have problems storing literal strings, so these
+  # getter/setter overrides wrap and unwrap those strings in hashes.
+  def description=(value)
+    if value.is_a?(String)
+      super(string: value)
+    else
+      super
+    end
+  end
+
+  def description
+    value = super
+
+    if value.is_a?(Hash) && value.key?(:string)
+      value.fetch(:string)
+    else
+      value
+    end
+  end
```

Comment:
Good plan: https://github.com/alphagov/publishing-api/commit/dafa1de03d8d4b4eab63b9eaa9c886f2816a078f

That field can hold other things, that aren't just strings, so I called the key `value`.

I also overrode `#attributes`. See the commit message for more info.

What do you think?